src/main/java/com/ericsson/ei/jmespath/IncompletePathFilterFunction.java

```diff
@@ -0,0 +1,214 @@
+package com.ericsson.ei.jmespath;
```

Comment:
Where is the unit tests for this class?

Reply:
com.ericsson.ei.jmespath.test;

Reply:
in src/test/java

Reply:
I guess these function is new implementations and then it should new unit tests be written for these new functions. or? And these new unit testes should then be included in this PR.

Reply:
I wrote new unit tests for the whole jmespath extension that I created. The 2 last tests in TestJmesPathInterface tests incomplitePathFilter.

Reply:
I created new test class and I moved my tests to it. It should be better now.